src/appleseed.studio/main/main.cpp

```diff
@@ -290,26 +290,26 @@ namespace
         // On Windows, there is a default message handler.
         if (g_previous_message_handler != nullptr)
         {
-            g_previous_message_handler(type, msg);
+            g_previous_message_handler(type, context, msg);
             return;
         }
 
         switch (type)
         {
           case QtDebugMsg:
-            fprintf(stderr, "Debug: %s\n", msg);
+            fprintf(stderr, "Debug: %s\n", msg.toLocal8Bit().constData());
```

Comment:
I'm not sure we want to use `toLocal8Bit()` / `fromLocal8Bit()`. Ideally appleseed would internally only handle UTF-8 strings.

Reply:
Could convert these to `toUtf8()` / `fromUtf8()` instead... I'm not exactly sure the implications of one or the other to be completley honest, the Local8Bit versions seemed safter but I could be off base on that assumption :)